packages/fuels-contract/src/script.rs

```diff
@@ -402,4 +463,344 @@ mod test {
             script_data[custom_input_offset..custom_input_offset + 2 * WORD_SIZE].to_vec();
         assert_eq!(custom_input, args[1]);
     }
+
+    #[cfg(test)]
+    mod creating_inputs_and_outputs {
```

Comment:
Me neither. Hate the indent this creates. But on the other hand, this allows me to import using `super::*` which is great for readability.

I think I'll leave this as is because of the import, seeing as I think I've seen other projects in fuel doing the same thing.